docs-website-src/content/blockers.md

```diff
@@ -74,6 +74,10 @@ You can discover many of these issues by downloading `elevate-cpanel` and runnin
 
 The following is a list of other known issues that could prevent your server's successful elevation.
 
+## Filesystem
+
+Since elevate needs to reboot your system multiple times as part of the upgrade process, we ensure that the command 'mount -a' succeeds successfully before allowing the elevation to proceed.  The reason for this is that we need to be able to trust that the filesystem remains the same between each reboot.
```

Comment:
Other instances of "elevate" in this document are written as ELevate, this one should match. I'd also make it read: the `mount -a` command